manpages/awesome.1.txt

```diff
@@ -46,6 +46,8 @@ OPTIONS
     Add a directory to the library search path.
 *-a*, *--no-argb*::
     Don't use ARGB visuals.
+*-m*, *--screen*:: 'off' or 'on'::
+    Use "off" to execute rc.lua before creating the screens.
```

Comment:
BTW @Veratil should confirm but I think the "the" is unneeded:

```suggestion
    Use "off" to execute rc.lua before creating screens.
```

Reply:
This is sort of a mixed bag here. Both are correct grammar, but carry slightly different feel of the word "screens".

After saying it a few times to myself, I do believe removing "the" carries the proper feel for the intention of what "screens" are.